source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js

```diff
@@ -243,32 +244,39 @@
                     this.calificationActionTooltip("Evaluar como responsable");
                     this.calificationActionClass("icon user");
                     this.calificationActionText("Evaluar");
+                    this.sortByPriority(0);
                     return;
                 } else if (this.state() == 1 || this.state() == 3) {
                     this.calificationActionText("Evaluar");
                     this.calificationActionClass("icon empresa");
                     this.calificationActionTooltip("Evaluar como empresa");
-                    return;
+                    this.sortByPriority(2);
+                    return
                 } else if (this.state() == 5) {
                     this.calificationActionText("Devolución");
                     this.calificationActionClass("icon user");
                     this.calificationActionTooltip("Hacer la devolución con el evaluado");
+                    this.sortByPriority(3);
                     return;
                 }
             } else if (!this.isResponsible && (this.state() == 4 || this.state() == 5)) {
                 this.calificationActionTooltip("Lista para devolucion");
                 this.calificationActionClass("icon view");
                 this.calificationActionText("Ver Evaluación");
+                this.sortByPriority(4);
                 return;
             } else if (!this.isResponsible && this.isEditable && this.state() != 6) {
                 this.calificationActionTooltip("Evaluar como evaluador");
                 this.calificationActionClass("icon user");
                 this.calificationActionText("Evaluar");
+                this.sortByPriority(1);
                 return;
             }
             this.calificationActionText("Ver Evaluación");
             this.calificationActionClass("icon view");
             this.calificationActionTooltip("Ver Evaluación");
+            this.sortByPriority(5);
```

Comment:
I think that using priorities to sort the actions is doing what you need to do in a long way: you're sorting by priorities because you know that each action as a unique priority and so actions will be sorted. What you actually wanted to do is sort actions -- is it possible that you sort actions?

Users are going to expect actions to be sorted on something that they can understand. If you sort by the action name, for example, that would be understandable, because users will be able to tell that actions are sorted or unsorted alphabetically by their name.

If you wish to give actions an "order" to be sorted, which is not alphabetical, that is fine too -- but that's a sort order rather than a priority. If the priority was built exactly for this, then maybe it could be named differently to reflect that. If it was not, then you're relying on your knowledge of the internals of the action models and that's breaking encapsulation.